### ISSUE TITLE
AddForm is dragable

### ISSUE THREAD

**Issue by bazzacad** [riktar/jkanban#25]
If I run the example page & hit the + button to call the "AddForm" option, the form is dragable

![image](https://user-images.githubusercontent.com/23222121/46317298-72147400-c587-11e8-926f-7c5dd7dcef9f.png)

**Comment by marcosrocha85** [comment]
@riktar We can solve that by adding form at footer instead of board itself.
```
this.addForm = function (boardID, formItem) {
            var board = self.element.querySelector('[data-id="' + boardID + '"] footer'); // was .kanban-drag
            board.appendChild(formItem);
            return self;
        };
```

**Comment by riktar** [comment]
resolved in #27
